.rubocop_todo.yml

```diff
@@ -1,32 +1,41 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2015-10-14 13:50:41 -0700 using RuboCop version 0.34.2.
+# on 2017-02-25 23:23:21 +0900 using RuboCop version 0.46.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# Offense count: 4
+# Offense count: 5
+# Configuration parameters: Include.
+# Include: **/Gemfile, **/gems.rb
+Bundler/OrderedGems:
+  Exclude:
+    - 'Gemfile'
+
+# Offense count: 3
 Metrics/AbcSize:
   Max: 27
 
 # Offense count: 1
-# Configuration parameters: CountComments.
-Metrics/ClassLength:
-  Max: 109
-
-# Offense count: 1
 Metrics/CyclomaticComplexity:
   Max: 7
 
-# Offense count: 16
+# Offense count: 18
 # Configuration parameters: CountComments.
 Metrics/MethodLength:
   Max: 22
 
-# Offense count: 2
+# Offense count: 3
 # Configuration parameters: EnforcedStyle, SupportedStyles.
+# SupportedStyles: format, sprintf, percent
 Style/FormatString:
   Exclude:
     - 'lib/googleauth/user_authorizer.rb'
     - 'lib/googleauth/web_user_authorizer.rb'
+
+# Offense count: 1
+# Configuration parameters: MinBodyLength.
+Style/GuardClause:
+  Exclude:
+    - 'lib/googleauth/web_user_authorizer.rb'
```

Comment:
https://github.com/google/google-auth-library-ruby/blob/master/lib/googleauth/web_user_authorizer.rb#L230-L237 is detected for this Cop. 

But I think we do not need to fix this warning.
If I would fix this warning, it might be more complicated.